hub/actions.py

```diff
@@ -71,7 +71,7 @@ def delete_related_objects(modeladmin, request, queryset):
                 n += 1
         modeladmin.message_user(
             request,
-            _("Successfully deleted %(count)d related objects.") % {
+            "Successfully deleted %(count)d related objects." % {
```

Comment:
Can we verify that `gettext()` imports are still needed

Reply:
according to the python documentation [here](https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#internationalization-in-python-code) `gettext()` still needs to be imported